impl/client/src/main/javascript/web/dojo/pentaho/common/FilterDialog.css

```diff
@@ -2,26 +2,110 @@
   display: none !important;
 }
 
+.responsive.pentaho-common-dialog.filter-dialog .dijitDialogPaneContent{
+  overflow: auto !important;
+}
+
 .filterDialogContainer {
   width: 560px;
 }
 
 .filterDialogDetailsContainer {
-  min-height: 250px;
+  min-height: 120px;
 }
 
 .filterDialogMatchContainer {
   height: 50px;
 }
 
+.filterPicklist{
```

Comment:
Formatting is off. Missing space before curly brace. In this one and all other new rules in this file.